src/locales/en_US.ts

```diff
@@ -1463,11 +1463,13 @@ const strings = {
   fiat_plugin_fetching_assets: 'Fetching supported assets',
   fiat_plugin_sell_cancelled: 'Sell order cancelled',
   fiat_plugin_finalizing_quote: 'Finalizing your exchange quote. Please wait as this may take up to a minute',
+  fiat_plugin_buy_cancelled: 'Buy order cancelled',
```

Comment:
Common misspelling. In American English, it's actually "canceled."

Recommend changing all these here and in the respective jsons. See `transaction_details_swap_order_cancel`

Reply:
https://www.merriam-webster.com/grammar/canceled-or-cancelled#:~:text=While%20both%20canceled%20and%20cancelled,more%20common%20in%20British%20English

Reply:
Alternatively, change `transaction_details_swap_order_cancel` to use two L's, since Americans don't really follow American spelling for this, anyway.

Either way, just make sure we're consistent.